src/Model/Contact.php

```diff
@@ -1197,8 +1199,20 @@ public static function photoMenu(array $contact, int $uid): array
 			$pm_url = 'message/new/' . $contact['id'];
 		}
 
+		if (in_array($contact['rel'], [contact::SHARING])) {
+			$mention_url = 'compose/0?body=!' . $contact['addr'];
+		} else {
+			$mention_url = 'compose/0?body=@' . $contact['addr'];
+		}
+
+
+		if (in_array($contact['rel'], [contact::SHARING])) {
+			$self = true;
+		}
```

Comment:
What is the meaning of this? It's the same condition as above, but for yet another purpose?